Assets/UI/Popups/greatpeoplepopup.lua

```diff
@@ -154,14 +154,13 @@ function ViewCurrent( data:table )
         textureOffsetY = 0;
         textureSheet = "GreatPeopleClass90";
       end
-      instance.ClassImage:SetTexture( textureOffsetX, textureOffsetY, textureSheet );
     end
```

Comment:
I would comment this if since the ClassImage is not used anymore and instead uncomment the next if

Reply:
I was hesitant to delete it and thought about commenting this. In the end I figured, the comment would clutter the code...
I'll still to the practice of rather keeping code than deleting it in the future.

Reply:
I meant commenting the whole `if`, from line 148 to line 158, sorry it was not clear

Reply:
Hope this is right now.